Formula/cqlkit.rb

```diff
@@ -0,0 +1,26 @@
+class Cqlkit < Formula
+  desc "CLI tool to export Cassandra query as CSV and JSON format"
+  homepage "https://github.com/tenmax/cqlkit"
+  url "https://github.com/tenmax/cqlkit/releases/download/v0.3.1/cqlkit-0.3.1.zip"
+  sha256 "f1d0cdbfa0780ee66c3b6e42d9eef1dd6b9c925918090ea2d4a547cb80dca4e8"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8"
+
+  def install
+    libexec.install %w[bin lib]
+    bin.install_symlink libexec+"bin/cql2cql"
+    bin.install_symlink libexec+"bin/cql2csv"
+    bin.install_symlink libexec+"bin/cql2json"
+  end
+
+  test do
+    output = shell_output("#{bin}/cql2cql -v")
+    assert_match /cql2cql version #{version}/, output
+    output = shell_output("#{bin}/cql2csv -v")
+    assert_match /cql2csv version #{version}/, output
+    output = shell_output("#{bin}/cql2json -v")
+    assert_match /cql2json version #{version}/, output
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I think we may need to install cassandra to test the cli tools. The new tests try to connect to localhost and the CLI tools will return query failed.